k8s/jiva/snapshot-cleanup.sh

```diff
@@ -42,8 +42,14 @@ delete_jiva_snapshot()
        exit 1;
     fi
 
-    snapshot_number_cmd='kubectl exec -it $ctrl_pod_name -n $pvc_namespace -- jivactl snapshot ls | grep -v ID | wc -l'
-    snapshot_name_cmd='kubectl exec -it $ctrl_pod_name -n $pvc_namespace -- jivactl snapshot ls | grep -v ID | tail -1'
+    snapshot_number_cmd="kubectl exec -it $ctrl_pod_name -n $pvc_namespace -- jivactl snapshot ls | grep -v ID | wc -l"
+    snapshot_name_cmd="kubectl exec -it $ctrl_pod_name -n $pvc_namespace -- jivactl snapshot ls | grep -v ID | tail -1"
+    
+    if [ $(eval $snapshot_number_cmd) -le $(($min_required_snapshot + $number_of_snapshot)) ]; then
+        echo "Error: 0 snapshot present for deletion"
+        echo "Error: You have requested to delete $number_of_snapshot. There are only $(eval $snapshot_number_cmd) snapshots are present. Minimum $min_required_snapshot snpshots should be present to perform deletion"
```

Comment:
`Error: You can initiate snapshot deletion only when the volume has more than $min_required_snapshot snapshots. There are only $(eval $snapshot_number_cmd) snapshots at the moment. You need not cleanup any more snapshots.`